yt_dlp/extractor/vrv.py

```diff
@@ -217,7 +217,7 @@ def _real_extract(self, url):
                 })
 
         thumbnails = []
-        for thumbnail in video_data.get('images', {}).get('thumbnails', []):
+        for thumbnail in video_data.get('images', {}).get('thumbnail', [])[0]:
```

Comment:
```suggestion
        for thumbnail in traverse_obj(video_data, ('images', 'thumbnail', ...)):
```

import `traverse_obj` from utils

Reply:
I've tried using that suggestion, but when trying to get the thumbnail, it fails with `'list' object has no attribute 'get'`, as thumbnail is an array of images. I don't use python much so excuse my ignorance 😅

Reply:
You may have removed the `...`?

Reply:
I had to use two `...` for the code to work correctly, if what I just commited seems right.

Reply:
oh, right. That seems correct